<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IpInfo - pythonexplainedto.me</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>221</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="IPLabel">
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="IpE"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="hostnameLabel">
      <property name="text">
       <string>Hostname</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="hostnameE"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="cityLabel">
      <property name="text">
       <string>City</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="cityE"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="regionLabel">
      <property name="text">
       <string>Region</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="regionE"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="countryLabel">
      <property name="text">
       <string>Country</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="countryE"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="locationLabel">
      <property name="text">
       <string>Location</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="locationE"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="orgLabel">
      <property name="text">
       <string>Org</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="orgE"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="postalLabel">
      <property name="text">
       <string>Postal</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="postE"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="timezoneLabel">
      <property name="text">
       <string>Timezone</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="timezE"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="getData">
      <property name="text">
       <string>Fetch Data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="remData">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
